### PR TITLE
adding card-2 design

### PR DESCRIPTION
**Pull request by Charan42**

### CODE REVIEW

**Comment by Charan42** [comment]
[Magasinet: Card-2 for magasinet HW task](https://trello.com/c/DlkENMuA/315-magasinet-card-2-for-magasinet-hw-task)

File: src/shared/components/card-2/Card-2.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+```
+
+### Regular card
+
+```html|span-3
+  <a href="#" class="hw-card-2">
+    <div class="hw-card-2__media">
+      <div class="hw-card-2__media-inner" style="background-image: url('https://bring-hedwig.s3.amazonaws.com/hedwig/assets/images/demo-image-1.png');"></div>
+    </div>
+    <div class="hw-card-2__body">
```
Comment (viggo) [inline src/shared/components/card-2/Card-2.md]: Label is missing

File: src/shared/components/card-2/card-2.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  }
+
+  &:hover {
+    &::after {
+      opacity: 1;
+    }
+  }
+  &:hover .hw-card-2__body {
+    background-color: var(--hw-color-gray-lighter);
+    border: 3px solid #C1C1C1;
```
Comment (viggo) [inline src/shared/components/card-2/card-2.css]: Border top should not be there on desktop. Border left should not be there on mobile.

File: src/shared/components/card-2/card-2.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  }
+
+  /**
+   * Elements
+   */
+
+  &__media {
+    object-fit: cover;
```
Comment (viggo) [inline src/shared/components/card-2/card-2.css]: `object-fit` is for img-tags. You're probably looking for `background-size`.

File: src/shared/components/card-2/card-2.css [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    overflow: hidden;
+    height: 190px;
+    flex-grow: 1;
+  }
```
Comment (viggo) [inline src/shared/components/card-2/card-2.css]: `__media-inner` should probably have height 100%. As it is now, the image does not show.

File: src/shared/components/card-2/card-2.css [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  &__body {
+    background-color: var(--hw-color-gray-lightest);
+    z-index: 1;
+    padding: var(--hw-spacing);
+    transition: all 0.1s ease;
+    border: 3px solid #f2f2f2;
+
+  }
+  &__label {
+    color: var(--hw-color-gray);
```
Comment (viggo) [inline src/shared/components/card-2/card-2.css]: Label is too big

File: src/shared/components/card-2/card-2.css [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    }
+    &:hover .hw-card-2__body {
+      border: 3px solid #C1C1C1;
+      border-left: 3px solid #E6E6E6;
+    }
+  }
+
+  @media(--large){
+    &__media {
+      max-height: 200px;
```
Comment (Charan42) [inline src/shared/components/card-2/card-2.css]: guess this max-height should be replaced with height and placed under .hw-card-2__media-inner class.
also in design, the card has two variants ( one with a bigger image size in desktop view ). a separate class should handle it.

**Pull request merged**